neurom/fst/tests/test_neuritefunc.py

```diff
@@ -213,6 +213,12 @@ def test_section_branch_orders():
                     (0, 1, 1,  # type 3, basal dendrite
                      0, 1, 1)) # type 2, axon
 
+def test_section_strahler_orders():
+    strahler_orders = list(_nf.section_strahler_orders(SIMPLE))
+    assert_allclose(strahler_orders,
+                    (2, 1, 1,  # type 3, basal dendrite
```

Comment:
I made a more complicated test based on the wikipedia article, and I'm getting differences in the Strahler number:
```
# SWC structure:
# index, type, x, y, z, radius, parent
# Recreation of:
# https://commons.wikimedia.org/wiki/File:Flussordnung_(Strahler).svg

 1 1  0 0 0 1 -1
 2 3  0 2 0 1  1
 3 3 -1 2 0 1  2
 4 3  0 3 0 1  2
 5 3 10 3 0 1  4
 6 3 10 2 0 1  5
 7 3  9 2 0 1  6
 8 3 11 2 0 1  6
 9 3 10 4 0 1  5
10 3  9 4 0 1  9
11 3 11 4 0 1  9
12 3  0 4 0 1  4
13 3 -1 4 0 1 12
14 3  0 5 0 1 12
15 3 -5 5 0 1 14
16 3 -5 6 0 1 15
17 3 -5 4 0 1 15
18 3  5 5 0 1 14
19 3  5 6 0 1 18
20 3  5 4 0 1 18
```
Loaded with:
```
>>> n = nm.load_neuron('path/to/above.swc')
>>> sectionfunc.strahler_order(n.neurites[0].root_node)
6
```
which is higher than 4 in the wikipedia image.  Are there different interpretations of the Strahler number?